src/front-end/src/components/EmailVerify.js

```diff
@@ -1,8 +1,16 @@
+/* This component is only displayed if the user hasn't verified their account.
+This is to ensure that only valid email addresses are used for account 
+creation. If a fraudulent email was used, the user won't be able to access
+the functionality that a verified user can.
+*/
 import React, { Component } from 'react';
 import { Button } from 'antd';
 import fire from '../firebase-init';
 import { resendVerificationEmail } from '../helper-methods';
 
+/* Obtains information about the logged in user and passes it 
+to the helper method so that the verification email can be resent
+*/
```

Comment:
Note that what I said about imperative tone does not apply to what you're doing here, which is providing a description of what the method does, in general. In fact, you could even use JSDoc in this case, by using the following format for your comment:

```suggestion
/** Obtains information about the logged in user and passes it 
to the helper method so that the verification email can be resent. */
```
This results in developers seeing a nice description of what the function does when they hover over it or use autocomplete in their IDE (as shown in screenshot below).

![Screen Shot 2021-03-15 at 5 49 10 PM](https://user-images.githubusercontent.com/21204917/111189821-c294b600-85b6-11eb-8fce-c417048881cd.png)

Reply:
Thanks for mentioning this, I wasn't aware that I could do that! Now that I know, I've added that into a couple of other places as well :)